.build

```diff
@@ -1,4 +1,5 @@
 CC = gcc
```

Comment:
Ok, I'll merge this now. I have more related to this that will come in but I'd rather avoid making this a cathedral